Gruntfile.js

```diff
@@ -1070,7 +1075,7 @@ module.exports = function(grunt) {
 
   grunt.registerTask('ci-e2e', 'Run e2e tests for CI', [
     'exec:start-webdriver',
-    'protractor:e2e-tests',
+    'protractor:e2e-tests-headless',
```

Comment:
I'd name this e2e-tests-ci after the scenario. Then we can make multiple changes for local vs ci without renaming.